docs/book/src/user/quick-start.md

```diff
@@ -793,15 +883,15 @@ export IBMPOWERVS_NETWORK_NAME=<your-capi-network-name>
 Please visit the [IBM Cloud provider] for more information.
 
 {{#/tab }}
-{{#tab Kubevirt}}
-
-A ClusterAPI compatible image must be available in your Kubevirt image library. For instructions on how to build a compatible image
-see [image-builder](https://image-builder.sigs.k8s.io/capi/capi.html).
+{{#tab KubeVirt}}
 
-To see all required Kubevirt environment variables execute:
 ```bash
-clusterctl generate cluster --infrastructure kubevirt --list-variables capi-quickstart
+export CAPK_GUEST_K8S_VERSION="v1.23.10"
+export CRI_PATH="/var/run/containerd/containerd.sock"
+export NODE_VM_IMAGE_TEMPLATE="quay.io/capk/ubuntu-2004-container-disk:${CAPK_GUEST_K8S_VERSION}"
+export IMAGE_REPO="k8s.gcr.io"
```

Comment:
can we switch this to the new registry, images should be already available in both (to be verified)

Reply:
I tried that several times and it's just not working. After looking at the code, I think it should only work from K8s v1.25+.

I also noticed that other providers just dropped this setting, so I opened a PR to drop it also in Kubevirt provider (kubernetes-sigs/cluster-api-provider-kubevirt#207) and I hope it will be included in the next version.

Reply:
We are using the new registry for Kubernetes >= 1.22

Reply:
+1 to your PR. I would definitely recommend not exposing IMAGE_REPO as an env var